.gitignore

```diff
@@ -6,7 +6,6 @@ id_rsa
 id_rsa.pub
 google-authz.json
 auth.js
-/index.js
```

Comment:
Why was this in here before? Why is auth.js?

Reply:
not exactly sure. There's nothing sensitive in those files.

Reply:
auth.js doesn't event exist